build/release.js

```diff
@@ -53,8 +54,8 @@ module.exports = function( Release ) {
 		 * committed before creating the tag.
 		 * @param {Function} callback
 		 */
-		generateArtifacts: function( callback ) {
-			Release.exec( "npm run build:all" );
+		generateArtifacts: async function( callback ) {
+			await buildDefaultFiles( { version: Release.newVersion } );
```

Comment:
Passes the version directly from jquery-release, which overrides any version creation in our builder.